SpatialGDK/Source/SpatialGDK/Public/Utils/EntityPool.h

```diff
@@ -25,7 +23,7 @@ class FTimerManager;
 
 DECLARE_LOG_CATEGORY_EXTERN(LogSpatialEntityPool, Log, All)
 
-DECLARE_DYNAMIC_MULTICAST_DELEGATE(FEntityPoolReadyEvent);
+DECLARE_DELEGATE(FEntityPoolReadyEvent);
```

Comment:
Should we make this a `DECLARE_MULTICAST_DELEGATE`? The user can do AddXXX functions so there are multiple handlers to this event.

Reply:
I made this a normal delegate because there was only one user AFAICT, and that seemed more explicit. Happy to change though

Reply:
That's fair, no one is stopping us from changing back if there's ever a need. I'll leave this up to you.